src/main/java/Lesson7/ExternalService/CurrencyCalculatorService.java

```diff
@@ -0,0 +1,9 @@
+package Lesson7.ExternalService;
+
+import Lesson7.Currency;
+
+public interface CurrencyCalculatorService {
+    double getSellRate(Currency currency);
```

Comment:
Wydaje mi się, że czegoś tu brakuje - jako argument podajesz jedną walutę. Jeżeli pójdziesz do losowego kantoru na świecie (nie tylko w PL) i zapytasz jaka jest cena EUR to w zależności od kraju odpowiedzą Ci w różnej walucie. 

W tej metodzie brakuje więc jako argumentu drugiej waluty - coś w stylu "from" i "to".

Abyś mógł odpowiedzieć na pytanie:

Ile PLN kosztuje EUR?
ile EUR kosztuje PLN?
ile PLN kosztuje $$
ile $$ kosztuje PLN
itd..

Reply:
bo bez tego drugiego parametru - skąd się mamy domyślić w jakiej walucie dostaniemy tego double , którego metoda zwraca? ;)